src/amr/messengers/hybrid_messenger.h

```diff
@@ -149,11 +149,11 @@ namespace amr
          * @brief see IMessenger::firstStep
          * @param model
          */
-        void firstNonRootStep(IPhysicalModel& model, SAMRAI::hier::PatchLevel& level,
-                              std::shared_ptr<SAMRAI::hier::PatchHierarchy> const& hierarchy,
-                              double time, double newCoarserTime) final
+        void firstStep(IPhysicalModel& model, SAMRAI::hier::PatchLevel& level,
```

Comment:
possible unnecessary change

Reply:
from the multiphysics integrator standpoint this operation is not necessarily reserved to non root levels, that's (for now) only the case for hybrid stuff but I can't say for other ones...